src/content/app/tools/vep/views/vep-submissions/listed-vep-submission/ListedVepSubmission.module.css

```diff
@@ -28,6 +28,7 @@
   grid-template-columns: [species-name] 300px [input-summary] 1fr [status] auto;
   column-gap: var(--standard-gutter);
   align-items: center;
+  line-height: 30px;
```

Comment:
Is this for setting the height of the submission box? `line-height` is such a non-obvious way of achieving this; and it will have undesirable consequences if text wraps into several lines. Wasn't min-height capable of achieving the desired effect?

Reply:
I can see that you've updated this line; but before your latest commit had a chance to deploy, I saw a good example demonstrating why using line-height would be bad.

https://github.com/user-attachments/assets/e966ffda-76e5-4e0b-96a0-975c2af0c66c